config/locales/en.yml

```diff
@@ -403,6 +403,7 @@ en:
       header: "%{title} Dependencies"
     gem_members:
       authors_header: Authors
+      single_owner_not_using_mfa_warning_show : "Please consider <a href="/settings/edit">enabling MFA</a> to keep your account secure."
```

Comment:
I am pretty sure you need to use `.._html` key for raw HTML to work. Double quotes should not be needed, the yaml is invalid because of it.